### DISCUSSION TITLE
About the code snippets

### DISCUSSION THREAD

**Discussion by Himan10** [Sampreeth1608/stockAutomation#3] [General]
## File: sample_code.gs

This sample code is just for testing the workflow of the project, checking if it's working as per the requirements and things I discussed with Sampreeth. It has nothing to do with the actual code but we don't have to rewrite another code in order to achieve the same, just some editions and then, we can use it as a final product

This code snippet contains 4 functions : 
1. **onSheet1()** : Write the `GOOGLEFINANCE` function on the very first cell of the google sheet, so that it can fetch the data and write the values in rows & columns without user interaction or user-defined script. 

2. **onSheet2()** : Write the data which we got by using the above function on the sheet this function has access to, The main thing here is, the function is responsible to write only 4 values after every single minute pass by.  This is again for testing purposes, these values in the function and whatever the project owner wants can be easily changed in the future. 

3. **trigger()**: Create a time-driven trigger that runs after every minute. Here, the function I used for a trigger is `onSheet2()`

4. **deleteTrigger()**: Delete all the running triggers, this will help to reduce the unnecessary load of triggers if they can't perform well.

**Comment by Himan10** [comment]
You can check out this spreadsheet : [Link to Google Sheet](https://docs.google.com/spreadsheets/d/1qutpCUrPPgjZY9ngcuFqp_Ryjx39BO0o20qQdc_Il6E/edit#gid=23402987) while running the code snippet mentioned above or use it to if you run this code snippet live on [google scripts](script.google.com)

**Comment by Sampreeth1608** [comment]
Thanks bro
